File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/examples-js/DatabaseSecurityAlertListByDatabase.js
const { SqlManagementClient } = require("@azure/arm-sql");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Gets a list of database's security alert policies.
 *
 * @summary Gets a list of database's security alert policies.
 * x-ms-original-file: specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/examples/DatabaseSecurityAlertListByDatabase.json
 */
async function getTheDatabaseThreatDetectionPolicies() {
  const subscriptionId = "00000000-1111-2222-3333-444444444444";
  const resourceGroupName = "securityalert-6852";
  const serverName = "securityalert-2080";
  const databaseName = "testdb";
  const credential = new DefaultAzureCredential();
  const client = new SqlManagementClient(credential, subscriptionId);
  const resArray = new Array();
  for await (let item of client.databaseSecurityAlertPolicies.listByDatabase(
    resourceGroupName,
    serverName,
    databaseName
  )) {
    resArray.push(item);
  }
  console.log(resArray);
}

getTheDatabaseThreatDetectionPolicies().catch(console.error);

File: specification/sql/resource-manager/Microsoft.Sql/stable/2021-11-01/examples-java/DatabaseSecurityAlertListByDatabase.java

import com.azure.core.util.Context;

/** Samples for DatabaseSecurityAlertPolicies ListByDatabase. */
public final class Main {
    /*
     * x-ms-original-file:
     * specification/sql/resource-manager/Microsoft.Sql/stable/2021-11-01/examples/DatabaseSecurityAlertListByDatabase.
     * json
     */
    /**
     * Sample code: Get the database's threat detection policies.
     *
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void getTheDatabaseSThreatDetectionPolicies(com.azure.resourcemanager.AzureResourceManager azure) {
        azure.sqlServers().manager().serviceClient().getDatabaseSecurityAlertPolicies()
            .listByDatabase("securityalert-6852", "securityalert-2080", "testdb", Context.NONE);
    }
}
